apps/api/src/main.ts

```diff
@@ -77,7 +69,7 @@ async function initializeNestApp() {
   app.enableCors({
     credentials: true,
     origin: [
-      'http://localhost:3000',
+      'http://localhost:3025',
```

Comment:
Change this back to 3000 otherwise we will have to alter a lot of docs

Reply:
Default post is never a good idea, it's best to change the docs

Reply:
I think i dont see any docs for this package yet. So maybe it wont harm to put forward this change